pkg/syncer/syncertest/fake/client.go

```diff
@@ -380,11 +380,6 @@ func (c *Client) Delete(_ context.Context, obj client.Object, opts ...client.Del
 		return newConflictingResourceVersion(id, obj.GetResourceVersion(), cachedObj.GetResourceVersion())
 	}
 
-	// Copy latest values back to input object
```

Comment:
Please add a comment with a link to the client code for delete to show that it doesn't use the `Into` like the other methods.  Pick a tag version to pin to when linking to github.

Reply:
Sounds good, done.